robottelo/host_helpers/satellite_mixins.py

```diff
@@ -310,6 +310,33 @@ def validate_pulp_filepath(
         return result.stdout
 
 
+class SetupHelpers:
```

Comment:
Just setup Helpers doents look good, again people might dump everything here, since every test has helpers for setup purpose mostly.

Name it in way that is purposeful and targetting specific need something like `ProvisioningSetup` or so. The other classes we have are `ContentInfo, Factories, SystemInfo, EnablePluginsSatellite` that targets the specifc classification.

Reply:
@jyejare  I think its better to have generic here, if we're going to add specific classes for component specific helpers then we'll end up inheriting SatelliteMixins class from so many classes, so SetupHelpers makes more sense to move setup specific helpers under it